### PR TITLE
feat(eventing): add target volumeid to nvme path events

### PR DESCRIPTION
**Pull request by datacore-vvarakantham**
Added target(volumeid) to name path events after obtaining it from nqn.

### CODE REVIEW

**Review by w3aman — APPROVED** [review]

File: control-plane/agents/src/bin/ha/node/server.rs [hunk 1]
```diff
@@ -432,10 +432,23 @@ impl ParsedUri {
 
 impl EventWithMeta for NodeAgentSvc {
     fn event(&self, event_action: EventAction, meta: EventMeta) -> EventMessage {
+        let volume_id = meta
```
Comment (tiagolobocastro) [inline control-plane/agents/src/bin/ha/node/server.rs]: Seems rather complicated, could we change the event fn to receive also optional nqn directly?

**Review by tiagolobocastro — APPROVED** [review]
Would be worth cherry-picking to 2.6 as well

File: control-plane/agents/src/bin/ha/node/server.rs (hunk 1 above)
Comment (datacore-vvarakantham) [reply control-plane/agents/src/bin/ha/node/server.rs]: We are using the Event trait here. We might have to add another event trait to pass the extra argument nqn.

Comment (tiagolobocastro) [reply control-plane/agents/src/bin/ha/node/server.rs]: A custom function here would probably suffice?

Comment (datacore-vvarakantham) [reply control-plane/agents/src/bin/ha/node/server.rs]: Added another private trait for this event.

**Review by tiagolobocastro — APPROVED** [review]

**Comment by datacore-vvarakantham** [comment]
bors merge

**Pull request merged**